src/components/ViewAdmin.vue

```diff
@@ -5,64 +5,75 @@ SPDX-License-Identifier: AGPL-3.0-or-later
 
 <template>
 	<div id="context_chat">
-		<NcSettingsSection :name="t('context_chat', 'Status')">
-			<NcNoteCard v-if="stats.initial_indexing_complete" show-alert type="success">
-				{{ t('context_chat', 'The initial indexing run finished at: {date}.', {date: showDate(stats.intial_indexing_completed_at)}) }}
-			</NcNoteCard>
-			<NcNoteCard v-else type="warning">
-				{{ t('context_chat', 'The initial indexing is still running.') }}
-			</NcNoteCard>
-			<NcNoteCard v-if="stats.backend_available" show-alert type="success">
-				{{ t('context_chat', 'The Context Chat Backend app is installed and responsive.') }}
-			</NcNoteCard>
-			<NcNoteCard v-else type="warning">
-				{{ t('context_chat', 'The Context Chat Backend app is not installed or not responsing.') }}
-			</NcNoteCard>
-			<NcNoteCard v-if="stats.initial_indexing_complete && stats.eligible_files_count > stats.vectordb_document_counts['files__default'] * 1.2" type="warning">
-				{{ t('context_chat', 'Less files were indexed than expected. Only {percent}% files out of {eligibleCount} are in the VectorDB.', {percent: Math.round((stats.vectordb_document_counts['files__default'] / stats.eligible_files_count) * 100), eligibleCount: stats.eligible_files_count}) }}
-			</NcNoteCard>
-			<table>
-				<thead>
-					<tr>
-						<th>{{ t('context_chat', 'Content provider') }}</th>
-						<th>{{ t('context_chat', 'Queued documents') }}</th>
-						<th>{{ t('context_chat', 'Documents in vector database') }}</th>
+		<NcSettingsSection :name="t('context_chat', 'Context Chat')">
+			<NcSettingsSection :name="t('context_chat', 'Indexing Status')">
```

Comment:
let's do a `<h3>` here since the sub-heading should look a little smaller.
we can also do away with the extra padding in the left if the above change does not fix it already.